data/blog/electron-4-0.md

```diff
@@ -0,0 +1,69 @@
+
+In addition, Electron 4 includes changes to Electron-specific APIs. You can find a summary of the major changes in Electron 4 below; for the full list of changes, check out the [Electron v4.0.0 release notes](TODO_RELEASE_NOTES).
+
+## Breaking Changes
+
+### Single Instance Locking
+
+Previously, to make your app a Single Instance Application (ensuring that only one instance of your app is running at any given time), you could use the `app.makeSingleInstance()` method. Starting in Electron 4.0, you must use `app.requestSingleInstanceLock()` instead. The return value of this method indicates whether or not this instance of your application successfully obtained the lock. If it failed to obtain the lock, you can assume that another instance of your application is already running with the lock and exit immediately.
+
+For an example of using `requestSingleInstanceLock()` and information on nuanced behavior on various platforms, [see the documentation for `app.requestSingleInstanceLock()` and related methods](https://electronjs.org/docs/api/app#apprequestsingleinstancelock) and [the `second-instance` event](https://electronjs.org/docs/api/app#event-second-instance).
+
+### `win_delay_load_hook`
+
+When building native modules for windows, the `win_delay_load_hook` variable in the module's `binding.gyp` must be true (which is the default). If this hook is not present, then the native module will fail to load on Windows, with an error message like `Cannot find module`. [See the native module guide](https://electronjs.org/docs/tutorial/using-native-node-modules#a-note-about-win_delay_load_hook) for more information.
+
+### Accessing `electron.screen` in Renderer Processes
+
+Instead of accessing the [screen module](https://electronjs.org/docs/api/screen) in a renderer process via `require('electron').screen`, you should use `require('electron').remote.screen`.
+
+### Accessing `require` in Sandboxed Renderer Processes
+
+If your renderer process is sandboxed, instead of requiring modules with `require('module')`, you should use `require('electron').remote.require('module')`.
+
+## Deprecations
+
+The following breaking changes are planned for Electron 5.0, and thus are deprecated in Electron 4.0.
+
+### Node.js Integration Disabled for `nativeWindowOpen`-ed Windows
+
+Starting in Electron 5.0, child windows opened with the `nativeWindowOpen` option will always have Node.js integration disabled.
+
+### `webPreferences` Default Values
+
+When creating a new `BrowerWindow` with the `webPreferences` option set, the following `webPreferences` option defaults are deprecated in favor of new defaults listed below:
+
+<div class="markdown-table-container">
```

Comment:
```suggestion
<div class="table table-ruled table-full-width">
```

@BinaryMuse If want you can use existing styles for tables. What it looks you can see it below.

| Prev | New |
| ---- | --- |
|  ![prev](https://user-images.githubusercontent.com/24681191/49872719-9e1f6180-fe22-11e8-8179-c773ada0305b.jpg) | ![new](https://user-images.githubusercontent.com/24681191/49872723-a24b7f00-fe22-11e8-9cf6-385ec87a0440.jpg) |